gitlab/util.go

```diff
@@ -11,19 +11,24 @@ import (
 )
 
 var accessLevelNameToValue = map[string]gitlab.AccessLevelValue{
-	"guest":     gitlab.GuestPermissions,
-	"reporter":  gitlab.ReporterPermissions,
-	"developer": gitlab.DeveloperPermissions,
-	"master":    gitlab.MasterPermissions,
```

Comment:
As @roidelapluie commented, this should probably be kept for backward compatibility.